Chummer/customdata/Neon Anarchy House Rules/amend_qualities.xml

```diff
@@ -275,43 +304,19 @@
 		</quality>
 		<quality>
 			<name>Trust Fund I</name>
-			<required amendoperation="replace">
-				<oneof>
-					<quality>SINner (National)</quality>
-					<quality>SINner (Corporate)</quality>
-					<quality>SINner (Corporate Limited)</quality>
-				</oneof>
-			</required>
+			<hide />
 		</quality>
 		<quality>
 			<name>Trust Fund II</name>
-			<required amendoperation="replace">
-				<oneof>
-					<quality>SINner (National)</quality>
-					<quality>SINner (Corporate)</quality>
-					<quality>SINner (Corporate Limited)</quality>
-				</oneof>
-			</required>
+			<hide />
 		</quality>
 		<quality>
 			<name>Trust Fund III</name>
-			<required amendoperation="replace">
-				<oneof>
-					<quality>SINner (National)</quality>
-					<quality>SINner (Corporate)</quality>
-					<quality>SINner (Corporate Limited)</quality>
-				</oneof>
-			</required>
+			<hide />
 		</quality>
 		<quality>
 			<name>Trust Fund IV</name>
-			<required amendoperation="replace">
-				<oneof>
-					<quality>SINner (National)</quality>
-					<quality>SINner (Corporate)</quality>
-					<quality>SINner (Corporate Limited)</quality>
-				</oneof>
-			</required>
+			<hide />>
```

Comment:
Double closing `>>` almost certainly will cause problems. You try to find and remove any and all of these (I don't have time to do a full sweep myself right now). I'd recommend you use a text editor with automatic tag completion like VSCode if you struggle with this in the future.

Reply:
I believe this was solved in the latest commit.